src/log-dnslog.c

```diff
@@ -358,8 +356,13 @@ static OutputCtx *LogDnsLogInitCtx(ConfNode *conf)
     return output_ctx;
 }
 
+#endif /* !HAVE_RUST */
+
 void LogDnsLogRegister (void)
 {
+#ifdef HAVE_RUST
+    SCLogNotice("LogDnsLogger not implemented for Rust DNS.");
```

Comment:
will this print a notice log unconditionally for HAVE_RUST cases? If so I would prefer a warning only if the logger is enabled

Reply:
Oops, I got this wrong.